### PR TITLE
JSON_INFO.md Mutations documentation update. Removed default fields from mutations.

### PR DESCRIPTION
**Pull request by Soyweiser**
Removed default time = 0 fields. Updated json documentation. (please check for errors). I didn't add attack definitions as I think these are still under development right?

### CODE REVIEW

File: data/json/mutations.json [hunk 1]
```diff
@@ -630,7 +630,7 @@
         "points" : -2,
         "description" : "You have problems with eating meat.  It's possible for you to eat it, but you will suffer morale penalties due to nausea.",
         "starting_trait" : true,
-        "//" : "Intolerances are presumed a psychological issue unless stated otherwise, and cancelled by digestive mutations, which force the issue.",
+        "note" : "Intolerances are presumed a psychological issue unless stated otherwise, and cancelled by digestive mutations, which force the issue.",
```
Comment (BevapDin) [inline data/json/mutations.json]: The "//" seems to be better, it's used (nearly) everywhere else and our JSON parser has special code to ignore repeated members of that name.

File: doc/JSON_INFO.md [hunk 2]
```diff
@@ -376,22 +376,34 @@ Mods can modify this via "add:traits" and "remove:traits".
 "description": "Nothing gets you down!" // In-game description
 "starting_trait": true, // Can be selected at character creation (default: false)
 "valid": false,      // Can be mutated ingame (default: true)
+"purifiable": false, //Cannot be purified (default: true)
```
Comment (BevapDin) [inline doc/JSON_INFO.md]: Misleading description: the "purifiable" member defines whether the mutation can be purified, *not* whether it cannot be purified. Compare to the line above.

File: doc/JSON_INFO.md [hunk 3]
```diff
@@ -401,7 +413,15 @@ Mods can modify this via "add:traits" and "remove:traits".
...
-]
+],
+"active" : true, //When set the mutation is an active mutation that the player needs to activate (default: false)
+"starts_active" : true, //When true, this 'active' mutation starts active (default: false, requires 'active')
+"cost" : 8, // Cost to activate this mutation. Needs one of the hunger, thirst, or fatigue values set to true. (default: 0)
+"time" : 100, //Sets the amount of (turns * current player speed ) time units that need to pass before the cost is to be paid again. Needs to be higher than one to have any effect. (default: 0)
+"hunger" : true, //If true, activated mutation increases hunger by cost. (default: false)
+"thirst" : true, //If true, activated mutation increases thirst by cost. (default: false)
+"fatigue" : true, //If true, activated mutation increases fatigue by cost. (default: false)
+"note" : "Yes, I love documentation!", // Documentation of the mutation. Obviously ignored ingame.
```
Comment (BevapDin) [inline doc/JSON_INFO.md]: > Obviously ignored ingame

As is any other member that is not explicitly loaded, like "note2", "other-note", "more notes for you", "dfjgw0rsl2majskfjdsfor" or "//". Why are those members not mentioned as well? I suggest removing this line as it does not really help.

Maybe add a general note about the member "//" being ignored to the top of the file or somewhere similar.

Comment (Soyweiser) [inline doc/JSON_INFO.md]: I wanted to suggest a common way to set comments in json files. As a common one can provide benefits later (forex: somebody implements a system where json files are automatically converted to wiki pages).

Will change it.

**Pull request merged**